
public class test {
	public static void main(String[] args) {
		try {
			
		} catch (Exception e) {
			// TODO: handle exception
		} catch (ArithmeticException a) {
			// TODO: handle exception
		}
	}
}
